.coveragerc-py2

```diff
@@ -0,0 +1,10 @@
+[run]
+branch=True
+source=unasync
```

Comment:
This might be causing a problem for Python 2 coverage, might need a `[paths] source = src/unasync` entry too.